sort/oddeven.go

```diff
@@ -0,0 +1,30 @@
+package sort
+
+//Odd-Even Sort is a variation of bubble sort where the sorting is divided into
```

Comment:
Thanks. Couple of minor things.
The comment always start with the function, variable, etc name.
So it should be "OddEvenSort is a variation...."

From GoDoc Documentation:

> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.